bokehjs/src/coffee/core/layout/solver.coffee

```diff
@@ -3,17 +3,10 @@ import {Variable, Expression, Constraint, Operator, Strength, Solver as Constrai
 import {Events} from "../events"
 
 _constrainer = (op) ->
-  () =>
-    expr = Object.create(Expression.prototype)
-    Expression.apply(expr, arguments)
-    return new Constraint(expr, op)
+  (args) -> new Constraint(new Expression(args...), op)
 
 _weak_constrainer = (op) ->
-  () ->
-    args = [null]
-    for arg in arguments
-      args.push(arg)
-    new Constraint( new (Function.prototype.bind.apply(Expression, args)), op, Strength.weak )
+  (args) -> new Constraint(new Expression(args...), op, Strength.weak)
```

Comment:
If it works. I have no objection. I do recall some weirdness that required the bind.apply but I don't have anything specific, as @bryevdv said, I took the lead from him and @sccolbert.